src/DataProtection/docs/Set-AzDataProtectionMSIPermission.md

```diff
@@ -0,0 +1,210 @@
+---
+external help file:
+Module Name: Az.DataProtection
+online version: https://docs.microsoft.com/powershell/module/az.dataprotection/set-azdataprotectionmsipermission
+schema: 2.0.0
+---
+
+# Set-AzDataProtectionMSIPermission
+
+## SYNOPSIS
+Grants required permissions to the backup vault to configure backup
+
+## SYNTAX
+
+```
+Set-AzDataProtectionMSIPermission -BackupInstance <IBackupInstanceResource> -PermissionsScope <String>
+ -VaultName <String> -VaultResourceGroup <String> [-KeyVaultId <String>] [-Confirm] [-WhatIf]
```

Comment:
A resource group and a vault name is enough to identify a key vault, why we still need a KeyVaultId

Reply:
We need this because **resource group and vault name correspond to backup vault**, whereas **KeyVaultId** is a parameter used in a specific case of granting permissions in Postgres server scenario

Reply:
So there will be two KeyVault in `Postgres server scemario`?

Reply:
no, only one key vault and one backup vault. backup vault is by default used in all data protection commands
-VaultName -VaultResourceGroup - this inputs the backup vault for all scenarios
[-KeyVaultId ] - this inputs the key vault for Postgres server scenario